README.md

```diff
@@ -13,6 +13,34 @@ The __filter__ package is generally provided as a folder with source (`.java`) a
 
 A sample set of images are provided in an __"images"__ directory.
 
+## Creating an executable jar file
+The project is already set up using the gradle wrapper so we can just use that to create the jar file.
+1. Open up the CLI and change the directory to the projects root directory.
```

Comment:
Suggested rewrite:
1. Using the command line interface, change to the project's root directory.